nat-lab/run_local.py

```diff
@@ -82,13 +82,35 @@ def main() -> int:
         verify_setup_correctness()
 
     if not args.nobuild:
-        run_build_command("linux", args)
-        # Run windows tests on WinVM
-        if args.windows:
-            run_build_command("windows", args)
-        # Run nat-lab natively on macOS (TODO: Add windows support)
-        if args.o == "darwin":
-            run_build_command("darwin", args)
+        print("\u001b[33m")
+        print("|=======================================================|")
+        print("| WARNING! Running builds requires atleast 16GBs of RAM |")
+        print("|=======================================================|")
+        print("\u001b[0m")
+        try:
+            run_build_command("linux", args)
```

Comment:
Does this mean that it's gonna be always run twice, do we want that?

Reply:
So script works the same as it used to. It builds linux, if needed builds windows and darwin separately, since they all are built in different environments. Meaning that it will be ran twice. In CI it won't build anything, bcuz `--nobuild` flag is passed and binaries a fetched from artifacts.